### ISSUE TITLE
README vs vignette

### ISSUE THREAD

**Issue by rodlammers** [rodlammers/countyfloods#25]
I edited the README.Rmd file but didn't make changes to the vignette. Just an FYI.

**Comment by geanders** [comment]
Okay, so do we want to just edit the README and then wait until the very end to get the vignette to match it? (Unless the vignette gets really long, which I wouldn't expect, I think they can have the same content.) That sounds like a good plan to me.

**Comment by rodlammers** [comment]
Yep, that sounds good. I tend to forget things so I also wanted to add this issue as a reminder to myself.

**Comment by rodlammers** [comment]
Also, I think I've completed all the major changes we talked about. If you have some time to look at the package again and let me know if you are satisfied with the new functions that would be great. No rush though.

**Comment by geanders** [comment]
Fantastic, I should get a chance to take a look this week.
